mopro-ffi/src/app_config/android.rs

```diff
@@ -32,10 +32,20 @@ pub fn build() {
 
     install_ndk();
     for arch in target_archs {
-        build_for_arch(&arch, &build_dir, &bindings_out, &mode);
+        build_for_arch(arch, &build_dir, &bindings_out, &mode);
     }
 
-    write_kotlin_bindings(&bindings_out);
+    generate_bindings(
+        (manifest_dir + "/src/mopro.udl").as_str().into(),
+        None,
+        KotlinBindingGenerator,
+        Some(bindings_out.to_str().unwrap().into()),
+        None,
+        None,
+        false,
+    )
+    .expect("Failed to generate bindings");
```

Comment:
This is not something we should do in this PR, but in the future we should probably refactor to use `anyhow::Result` as the return type in all these functions (including `main`). In all the rust codebases i've worked in (which to be fair is only like 4) I've had to refactor to add `anyhow::Result` everywhere.

If there's an error case we want to handle explicitly we can always do something like

```rs
if let Err(error) = fallible_operation() {
    println!("Operation did not succeed because of reason: {}", error);
    std::process::exit(1);
}
```